extensions/noteplan-3/package.json

```diff
@@ -37,6 +37,24 @@
       "title": "File Extension",
       "default": "txt",
       "description": "The file extension you use for notes"
+    },
+    {
+      "name": "noteplanFlavour",
```

Comment:
Why flavour?
I'm not sure people would follow the language.
Let's try something that communicates to non-savvy users that this is where their app came from.
More on the lines of "Installation Source" or "Store" or "Distributor", etc.
What do you think?

Reply:
@pernielsentikaer how would you call the application source? Having seen many extensions by now

Reply:
Makes sense. And, I guess, it would be good to rename variables in code too

Reply:
@shakedlokits @pernielsentikaer I have renamed 'flavour' to "installation source" in user-facing text and also variable names, please have a look